cmd/integtest/main.go

```diff
@@ -29,6 +29,7 @@ var (
 	pgdir    = dir + "/pg"
 	pgrun    = dir + "/pgrun" // for socket file
 	pglog    = dir + "/pglog" // for log file
+	chain    = first(os.Getenv("CHAIN"), home+"/go/src/chain")
```

Comment:
I like first because it conveys the precedence. 🚲  could be firstNonEmpty or firstPresent?